src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -939,7 +945,7 @@ func Main(Context openruntimes.Context) openruntimes.Response {
 	case "json":
 		return Context.Res.Json(map[string]string{"type": "This is a JSON response"})
 	case "binary":
-		file, _ := os.Open("./destiny.png")
+		file, _ := os.Open("./file.png")
```

Comment:
In compiled languages not all files are copied, instead, only compilation output.
For this to work, you would need additional command during build step to copy your asset files.
Possible, but to keep it simple we could do: https://gobyexample.com/embed-directive

Let's check other compiled languages and see if they have something similar: C++, Dart, Dotnet, Java/Kotlin, Swift